mfhdf/test/tszip.c

```diff
@@ -903,8 +900,11 @@ test_szip_chunk_3d()
     int16         fill_value = 0; /* Fill value */
     comp_coder_t  comp_type;      /* to retrieve compression type into */
     comp_info     cinfo;          /* compression information structure */
-    int           num_errs = 0;   /* number of errors so far */
-    int           i, j, k;
+    int16         all_data[SDS_DIM0][SDS_DIM1][SDS_DIM2];
+    int16         out_data[SDS_DIM0][SDS_DIM1][SDS_DIM2];
```

Comment:
This was set as a global for some reason, which resulted in a TON of shadow warnings